jOOQ-examples/jOOQ-spring-boot-example/pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <groupId>org.jooq</groupId>
     <artifactId>jooq-spring-boot-example</artifactId>
-    <version>${org.jooq.version}</version>
+    <version>${jooq.version}</version>
```

Comment:
Hmm, I see, this change may make sense in general. But currently, all the examples use this property naming scheme...